TLM/TLM/Util/PriorityRoad.cs

```diff
@@ -429,18 +432,27 @@ public static class PriorityRoad {
             }
         }
 
-        private static int CountLanes(ushort segmentId, ushort nodeId, bool toward) {
-            ExtSegmentManager extSegmentManager = ExtSegmentManager.Instance;
-            return extSegmentManager.GetSortedLanes(
-                                segmentId,
-                                ref segmentId.ToSegment(),
-                                ExtSegmentManager.Instance.IsStartNode(segmentId, nodeId) ^ (!toward),
-                                LaneArrowManager.LANE_TYPES,
-                                LaneArrowManager.VEHICLE_TYPES,
-                                true).Count;
-        }
-        internal static int CountLanesTowardJunction(ushort segmentId, ushort nodeId) => CountLanes(segmentId, nodeId, true);
-        internal static int CountLanesAgainstJunction(ushort segmentId, ushort nodeId) => CountLanes(segmentId, nodeId, false);
+        /// <summary>
+        /// Count number of applicable lanes entering or leaving a segment via specific node.
+        /// </summary>
+        /// <param name="segmentId">The id of the segment to inspect.</param>
+        /// <param name="nodeId">The id of node where lanes should be counted.</param>
+        /// <param name="incoming">
+        /// If <c>true</c>, count lanes entering the segment from the junction.
+        /// If <c>false</c>, count lanes going from the segment to the junction.
+        /// </param>
+        /// <returns>Returns number of lanes matching the specified criteria.</returns>
+        private static int CountLanes(ushort segmentId, ushort nodeId, bool incoming) =>
+            segmentId.ToSegment().CountLanes(
+                nodeId,
+                LaneArrowManager.LANE_TYPES,
+                LaneArrowManager.VEHICLE_TYPES,
+                incoming);
+
+        internal static int CountLanesTowardJunction(ushort segmentId, ushort nodeId)
+            => CountLanes(segmentId, nodeId, false);
+        internal static int CountLanesAgainstJunction(ushort segmentId, ushort nodeId)
+            => CountLanes(segmentId, nodeId, true);
```

Comment:
Similar code in other managers uses naming such as `CountOutgoingLanes` and `CountIncomingLanes` - should I update these to match for sake of consistency?

Reply:
Does it sacrifice performance?
Specially when we are highlighting in UI thread?

Reply:
There's no alteration in terms of performance to prior code either way, assuming mono can do basic inlining.